flo-workflow/src/main/java/com/spotify/flo/context/InstrumentedContext.java

```diff
@@ -49,7 +50,7 @@
   /**
    * A listener for instrumented evaluation. See {@link InstrumentedContext} for more details.
    */
-  public interface Listener {
+  public interface Listener extends Closeable {
```

Comment:
Having a default implementation in `Listener` which does nothing?

Reply:
We can only override the declaration of the `close()` method, not provide an empty implementation, due to Listener being an interface itself. But it seems unnecessary. Whoever implements `Listener` in their implementation class they will be required to implement the `close()` method too.

Reply:
That you can have for an interface.

Reply:
Right, yes, we can do that.